Makefile

```diff
@@ -1,10 +1,23 @@
+.POSIX:
+
 VERCMD  ?= git describe --tags 2> /dev/null
 VERSION := $(shell $(VERCMD) || cat VERSION)
```

Comment:
These lines are not valid POSIX make, but you have added `.POSIX:`.. 

Both because of the presence of `?=`/`:=` and because of the `$(shell ...)` function.

Many of the following lines also use `?=` and `:=`.